### PR TITLE
DDCE-5019 Play 3.0 upgrade

### PR DESCRIPTION
**Pull request by garytate123**
# DDCE-5019

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist PR Raiser
- [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
- [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions 
- [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -23,16 +23,11 @@ lazy val microservice = Project(appName, file("."))
...
-    libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always,
     defaultSettings(),
     majorVersion := 2,
     PlayKeys.playDefaultPort := 8098,
     libraryDependencies ++= AppDependencies(),
-    scalacOptions ++= Seq(
-      "-Wconf:src=routes/.*:s",
-      "-Wconf:cat=unused-imports&src=views/.*:s"
-    )
+    scalacOptions += "-Wconf:src=routes/.*:s"
```
Comment (adageorg) [inline build.sbt]: Does this mean we don't compile the views any more?

Comment (garytate123) [reply build.sbt]: The views or html are only required for Frontend repos. I'm not sure why the views where added here in the first place.

When upgrading to play 3.0, you will find the following in the documentation...
// only required for frontends
scalacOptions += "-Wconf:cat=unused-imports&src=html/.*:s", 
// for all services
scalacOptions += "-Wconf:cat=unused-imports&src=routes/.*:s",

Comment (adageorg) [reply build.sbt]: we should have the unused imports bit still in there, then, shouldn't we?

Comment (garytate123) [reply build.sbt]: Yes, good shout.

**Review by adageorg — APPROVED** [review]

**Pull request merged**